proposals/apple-mdm/base-setup-clix.md

```diff
@@ -0,0 +1,145 @@
+# MDM Base Setup
+
+Note: these can all be namespaced behind something like `apple-mdm` if needed. But I would like to avoid creating an mdm vertical vs. an osquery vertical to create an integrated product.
+
+## Items of Note
+- A NO-OP command (setup apple-mdm) which are pure instruction gathering only
+- Setup integrated into fleetctl, not namespaced into a separate vertical (e.g. `fleetctl apple-mdm`). That might happen later, but not for setup.
+
+- TODO: Need link for apple's instructions on how to upload .req file
+- TODO: Need clearer instructions for who/how to contact fleet
+- TODO: Need to communicate some expectation of time to receive fleet .p7m file back
+
+## fleetctl setup
+
+```
+$ fleetctl setup
+NAME:
+   fleetctl setup - Set up a Fleet instance or MDM // UPDATED
+
+USAGE:
+   fleetctl setup [options]
+   fleetctl setup apple-mdm // NEW
+   fleetctl setup apns // NEW
+   fleetctl setup scep // NEW
+   
+SUBCOMMANDS:
+  apple-mdm Instructions for enabling MDM for Apple devices
```

Comment:
I'm not a bit fan of having another subcommand that essentially prints help. That's what `-h,--help` are for. The instructions for setting up apple mdm should be part of the help text for `fleetctl setup`.

Reply:
What would you propose the `fleetctl setup` text look like in that instance?

Reply:
Perhaps have a section called `DESCRIPTION` lie
```
$ fleetctl setup --help
NAME:
...

USAGE:
...

DESCRIPTION:
   Instructions for setting up MDM for Apple devices:

   1. Run "fleetctl setup apns" ...
   2. Run "fleetctl setup scep" ...
   3. Set FLEET_APPLE_MDM_ENABLE=1 in your fleet server ...

   For more information on how to configure Fleet, visit https://fleetdm.com/docs/deploying/configuration#options

```